src/circuit.rs

```diff
@@ -459,8 +459,8 @@ mod tests {
       &params2,
       ro_consts1,
       ro_consts2,
-      &expect!["9817"],
```

Comment:
1. what's the logic behind the general nova circuit's constraint count changing?
2. can you please also update NUM_CONS_VERIFIER_CIRCUIT_PRIMARY ?

Reply:
Oh this is funky, I must have messed up the last rebase. These constraint counts shouldn't change, I can fix it real quick.